sources/web/datalab/static/datalab.js

```diff
@@ -102,7 +105,9 @@ define([
   'static/notebook-app',
   'static/notebook-list',
   'static/websocket'
-], function(ipy, events, dialog, utils, security, appbar, editapp,
+], function(ipy,     events, dialog, utils, security, appbar, editapp,
             minitoolbar, notebookapp, notebooklist, websocket) {
-  initializeDataLab(IPython, events, dialog, utils, security);
+     initializeDataLab(
+            IPython, events, dialog, utils, security, appbar, editapp,
```

Comment:
while at it, can we change `IPython` to `ipy` for consistency? `IPython` is the global variable, we shouldn't be passing that.